SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialGameInstance.h

```diff
@@ -59,6 +59,9 @@ class SPATIALGDK_API USpatialGameInstance : public UGameInstance
 	void HandleOnConnectionFailed(const FString& Reason);
 	void HandleOnPlayerSpawnFailed(const FString& Reason);
 
+	void HandleOnSimulatedPlayerTravelFailure(UWorld* World, ETravelFailure::Type TravelType, const FString& Reason);
```

Comment:
Believe these could be static functions?

Reply:
I've checked and it seems so; we can also go one step further and make them free functions in the .cpp file as they aren't really a part of the GameInstance, but this is up to you.

Reply:
They could be free as long as you can bind them to the delegates. Otherwise I am fine with static.